cmd/util/cmd/epochs/cmd/recover_test.go

```diff
@@ -77,30 +75,34 @@ func TestRecoverEpochHappyPath(t *testing.T) {
 		currEpoch := rootSnapshot.Epochs().Current()
 		finalView, err := currEpoch.FinalView()
 		require.NoError(t, err)
+		targetEndTime, err := currEpoch.TargetEndTime()
+		require.NoError(t, err)
 
+		// epoch counter
+		require.Equal(t, cadence.NewUInt64(flagEpochCounter), decodedValues[0])
 		// epoch start view
-		require.Equal(t, decodedValues[0], cadence.NewUInt64(finalView+1))
+		require.Equal(t, cadence.NewUInt64(finalView+1), decodedValues[1])
 		// staking phase end view
-		require.Equal(t, decodedValues[1], cadence.NewUInt64(finalView+flagNumViewsInStakingAuction))
+		require.Equal(t, cadence.NewUInt64(finalView+flagNumViewsInStakingAuction), decodedValues[2])
 		// epoch end view
-		require.Equal(t, decodedValues[2], cadence.NewUInt64(finalView+flagNumViewsInEpoch))
+		require.Equal(t, cadence.NewUInt64(finalView+flagNumViewsInEpoch), decodedValues[3])
 		// target duration
-		require.Equal(t, decodedValues[3], cadence.NewUInt64(flagTargetDuration))
+		require.Equal(t, cadence.NewUInt64(targetEndTime+flagRecoveryEpochTargetDuration), decodedValues[4])
 		// target end time
 		expectedTargetEndTime, err := rootSnapshot.Epochs().Current().TargetEndTime()
 		require.NoError(t, err)
```

Comment:
same calculation as in the implementation, which I think is bugged

Reply:
[`27150c1` (#6424)](https://github.com/onflow/flow-go/pull/6424/commits/27150c1c2ab0fc6cf20490bf62a4f0778332681b)